walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -14,11 +14,12 @@
 
 package com.google.sps;
 
-import java.util.ArrayList;
+import static java.util.Arrays.*;
```

Comment:
Please replace this start import with more granular and specific imports

Reply:
Ack